src/mbgl/map/transform_state.cpp

```diff
@@ -133,21 +133,20 @@ ViewportMode TransformState::getViewportMode() const {
 #pragma mark - Camera options
 
 CameraOptions TransformState::getCameraOptions(const EdgeInsets& padding) const {
-    CameraOptions camera;
-
-    if (padding.isFlush()) {
-        camera.center = getLatLng();
-    } else {
+    const LatLng center = [&]() {
```

Comment:
why do we need this self-invoking lambda?

Reply:
I wanted to resolve `center` value in a simple way - I guess it makes it less readable with the lambda :cry: